Mathlib/Lean/Meta/Simp.lean

```diff
@@ -133,10 +133,10 @@ end Simp
 def simpTheoremsOfNames (lemmas : List Name := []) (simpOnly : Bool := false) :
     MetaM SimpTheorems := do
   lemmas.foldlM (·.addConst ·)
-    (if simpOnly then
```

Comment:
Lifting `←` out of `if`: this is an error now, to avoid the footgun of unexpected computation.